### PR TITLE
Add option to initialize python functions.

### PR DESCRIPTION
**Pull request by taeold**
Add Python as an option for `firebase init functions`.

The python choice is guarded behind the `pythonfunctions` experiment. You can enable the experiment with `firebase experiments:enable pythonfunctions`.

This PR is a fork of https://github.com/firebase/firebase-tools/pull/4653. With this PR, https://github.com/firebase/firebase-tools/pull/4653 is now obsolete.

### CODE REVIEW

**Comment by taeold** [comment]
Current dependencies on/for this PR:
* master
  * **PR #5419** <a href="https://app.graphite.dev/github/pr/firebase/firebase-tools/5419" target="_blank">Graphite</a> 
    * **PR #5422** <a href="https://app.graphite.dev/github/pr/firebase/firebase-tools/5422" target="_blank">Graphite</a> 
      * **PR #5423** <a href="https://app.graphite.dev/github/pr/firebase/firebase-tools/5423" target="_blank">Graphite</a> 
        * **PR #5464** <a href="https://app.graphite.dev/github/pr/firebase/firebase-tools/5464" target="_blank">Graphite</a> 
          * **PR #5465** <a href="https://app.graphite.dev/github/pr/firebase/firebase-tools/5465" target="_blank">Graphite</a>  👈
        * **PR #5462** <a href="https://app.graphite.dev/github/pr/firebase/firebase-tools/5462" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/firebase/firebase-tools/5465?utm_source=stack-comment).

**Comment by taeold** [comment]
Current dependencies on/for this PR:
* master
  * **PR #5419** <a href="https://app.graphite.dev/github/pr/firebase/firebase-tools/5419" target="_blank">Graphite</a> 
    * **PR #5422** <a href="https://app.graphite.dev/github/pr/firebase/firebase-tools/5422" target="_blank">Graphite</a> 
      * **PR #5423** <a href="https://app.graphite.dev/github/pr/firebase/firebase-tools/5423" target="_blank">Graphite</a> 
        * **PR #5464** <a href="https://app.graphite.dev/github/pr/firebase/firebase-tools/5464" target="_blank">Graphite</a> 
          * **PR #5465** <a href="https://app.graphite.dev/github/pr/firebase/firebase-tools/5465" target="_blank">Graphite</a>  👈
        * **PR #5462** <a href="https://app.graphite.dev/github/pr/firebase/firebase-tools/5462" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/firebase/firebase-tools/5465?utm_source=stack-comment).

File: templates/init/functions/python/main.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+from firebase_functions import https
+from firebase_admin import initialize_app
+
+# initialize_app()
+#
+#
+# @https.on_request()
+# def on_request_example(req: https.Request) -> https.Response:
+#     return https.Response("Hello world!")
```
Comment (colerogers) [inline templates/init/functions/python/main.py:13]: newline at eof

**Review by colerogers — APPROVED** [review]

**Comment by taeold** [comment]
⏳ This pull request is set to merge as part of a Graphite merge job
Stack job ID: aDBFOYKNmnzR1pnl7vJM.
See details on [graphite.dev](https://app.graphite.dev/github/pr/firebase/firebase-tools/5465)

**Pull request merged**